### DISCUSSION TITLE
Welcome to cookiecutter-uv Discussions!

### DISCUSSION THREAD

**Discussion by fpgmaas** [fpgmaas/cookiecutter-uv#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alherm7** [comment]
Hey Florian.
why are you not adding prettier to your cookiecutter-uv template as you have done with your cookiecutter-poetry template? I guess there is a reason, but it would be nice to understand. :) Thanks!
